// styled-config/lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "theme", {
  enumerable: true,
  get: function get() {
    return _theme.theme;
  }
});
Object.defineProperty(exports, "border", {
  enumerable: true,
  get: function get() {
    return _utils.border;
  }
});
Object.defineProperty(exports, "borderRadius", {
  enumerable: true,
  get: function get() {
    return _utils.borderRadius;
  }
});
Object.defineProperty(exports, "boxShadow", {
  enumerable: true,
  get: function get() {
    return _utils.boxShadow;
  }
});
Object.defineProperty(exports, "colors", {
  enumerable: true,
  get: function get() {
    return _utils.colors;
  }
});
Object.defineProperty(exports, "fontFamily", {
  enumerable: true,
  get: function get() {
    return _utils.fontFamily;
  }
});
Object.defineProperty(exports, "fontSize", {
  enumerable: true,
  get: function get() {
    return _utils.fontSize;
  }
});
Object.defineProperty(exports, "fontWeight", {
  enumerable: true,
  get: function get() {
    return _utils.fontWeight;
  }
});
Object.defineProperty(exports, "height", {
  enumerable: true,
  get: function get() {
    return _utils.height;
  }
});
Object.defineProperty(exports, "margin", {
  enumerable: true,
  get: function get() {
    return _utils.margin;
  }
});
Object.defineProperty(exports, "padding", {
  enumerable: true,
  get: function get() {
    return _utils.padding;
  }
});
Object.defineProperty(exports, "screenSize", {
  enumerable: true,
  get: function get() {
    return _utils.screenSize;
  }
});
Object.defineProperty(exports, "width", {
  enumerable: true,
  get: function get() {
    return _utils.width;
  }
});

var _theme = require("./theme");

var _utils = require("./utils");
