// styled-config/lib/theme/alert.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.alert = void 0;

var _colors = require("./colors");

var _utils = require("./utils");

var alert = {
  colors: {
    primary: {
      text: _colors.colors.blueAlertText,
      background: _colors.colors.blueAlertBackground,
      border: _colors.colors.blueAlertBorder,
      borderTop: _colors.colors.blueAlertBorderTop,
      hover: _colors.colors.blueAlertAHoverFocus
    },
    secondary: {
      text: _colors.colors.grayAlertText,
      background: _colors.colors.grayAlertBackground,
      border: _colors.colors.grayAlertBorder,
      borderTop: _colors.colors.grayAlertBorderTop,
      hover: _colors.colors.grayAlertAHoverFocus
    },
    success: {
      text: _colors.colors.greenAlertText,
      background: _colors.colors.greenAlertBackground,
      border: _colors.colors.greenAlertBorder,
      borderTop: _colors.colors.greenAlertBorderTop,
      hover: _colors.colors.greenAlertAHoverFocus
    },
    danger: {
      text: _colors.colors.redAlertText,
      background: _colors.colors.redAlertBackground,
      border: _colors.colors.redAlertBorder,
      borderTop: _colors.colors.redAlertBorderTop,
      hover: _colors.colors.redAlertAHoverFocus
    },
    warning: {
      text: _colors.colors.yellowAlertText,
      background: _colors.colors.yellowAlertBackground,
      border: _colors.colors.yellowAlertBorder,
      borderTop: _colors.colors.yellowAlertBorderTop,
      hover: _colors.colors.yellowAlertAHoverFocus
    },
    info: {
      text: _colors.colors.cyanAlertText,
      background: _colors.colors.cyanAlertBackground,
      border: _colors.colors.cyanAlertBorder,
      borderTop: _colors.colors.cyanAlertBorderTop,
      hover: _colors.colors.cyanAlertAHoverFocus
    },
    light: {
      text: _colors.colors.gray100AlertText,
      background: _colors.colors.gray100AlertBackground,
      border: _colors.colors.gray100AlertBorder,
      borderTop: _colors.colors.gray100AlertBorderTop,
      hover: _colors.colors.gray100AlertAHoverFocus
    },
    dark: {
      text: _colors.colors.gray800AlertText,
      background: _colors.colors.gray800AlertBackground,
      border: _colors.colors.gray800AlertBorder,
      borderTop: _colors.colors.gray800AlertBorderTop,
      hover: _colors.colors.gray800AlertAHoverFocus
    },
    default: {
      text: _colors.colors.grayAlertText,
      background: _colors.colors.grayAlertBackground,
      border: _colors.colors.grayAlertBorder,
      borderTop: _colors.colors.grayAlertBorderTop,
      hover: _colors.colors.grayAlertAHoverFocus
    }
  },
  margin: {
    bottom: '1rem'
  },
  padding: {
    default: '0.75rem 1.25rem',
    dismissibleRight: '4rem'
  },
  border: {
    default: '1px solid transparent'
  },
  borderRadius: _utils.utils.borderRadius,
  fontWeight: {
    default: '700'
  }
};
exports.alert = alert;
